test-files/cmd/slow/NodeDensityTilesCmdTest/output-cropped.geojson

```diff
@@ -1,4 +1,4 @@
-{"generator":"Hootenanny","type":"FeatureCollection","bbox": [-77.049, 38.886, -77.028, 38.901],"import_url":"hootapidb://hoot@localhost:5432/hoot/DcTigerRoads;osmapidb://hoot@localhost:5432/osmapi_test","features": [
+{"generator":"Hootenanny","type":"FeatureCollection","bbox": [-77.049, 38.886, -77.028, 38.901],"source":"hootapidb:268;osmapidb:osmapi_test","features": [
```

Comment:
@brianhatchl here is the updated GeoJSON using the `source` property and `<schema>:<map id>` value.